cargo-dist/src/tasks.rs

```diff
@@ -3047,11 +3047,12 @@ impl InstallReceipt {
             // The actual value of $CARGO_HOME isn't known until inside the
             // install script, so we write out a temporary value that the
             // install script will replace with the real one later.
-            InstallPathStrategy::CargoHome => "AXO_CARGO_HOME".to_owned(),
+            InstallPathStrategy::CargoHome
+            | InstallPathStrategy::EnvSubdir {
+                env_key: _,
+                subdir: _,
+            } => "AXO_INSTALL_PREFIX".to_owned(),
             InstallPathStrategy::HomeSubdir { subdir } => format!("$HOME/{}", subdir),
-            InstallPathStrategy::EnvSubdir { env_key, subdir } => {
-                format!("${}/{}", env_key, subdir)
-            }
         };
```

Comment:
i would love if this PR was just making us unconditionally put AXO_INSTALL_PREFIX in and then all the paths in the shell and powershell installer do the find-replace. future-proof for all installer path features (as-is i'm not sure this would handle the env-var that forces the install dir).

Reply:
Seems reasonable; let me look into that.